pkgs/applications/networking/sniffers/wireshark/default.nix

```diff
@@ -24,13 +25,19 @@ stdenv.mkDerivation {
 
   buildInputs = [
     bison flex perl pkgconfig libpcap lua5 openssl libgcrypt gnutls
-    geoip libnl c-ares python libcap glib
+    geoip libnl c-ares python libcap glib zlib
```

Comment:
Is `zlib` is added unconditionally here. Is that really necessary? Or is that dependency required only by the qt build variant, maybe?